ui/shared/components/panel/search/FrequencyFilter.jsx

```diff
@@ -145,9 +145,9 @@ const callsetChange = (onChange, initialValues) => val => onChange(
   { ...initialValues, [THIS_CALLSET_FREQUENCY]: val, [SV_CALLSET_FREQUENCY]: val },
 )
 
-const freqChange = (onChange, initialValues) => val => onChange(FREQUENCIES.filter(
-  ({ name }) => name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY,
-).reduce((acc, { name }) => ({ ...acc, [name]: val }), initialValues || {}))
+const freqChange = (onChange, initialValues) => val => onChange(FREQUENCIES.reduce((acc, { name }) => ({
+  ...acc, [name]: name !== THIS_CALLSET_FREQUENCY && name !== SV_CALLSET_FREQUENCY ? val : initialValues[name],
+}), initialValues || {}))
```

Comment:
the reported issue with the AC in the header not updating occurred when a saved search was selected that still had a freq filter applied for a deprecated population (i.e. exac). The update did nto change those populations so the AC was not the same for all populations and thus it was not showing in the header. This fixes that issue by updating the filter on change to only include the currently supported/ visible to user populations